imaginedsf-custom-theme/header.php

```diff
@@ -12,10 +12,6 @@
                     <img src="<?php echo get_template_directory_uri() . '/images/logo.svg'; ?>" alt="Imagined San Francisco">
                 </a>
                 <div class="subflex">
-                    <form class="control has-icons-left search-an-address">
-                        <input class="input is-small" type="text" placeholder="search an address...">
-                        <span class="icon is-small is-left"><i class="fa fa-search"></i></span>
-                    </form>
```

Comment:
This look good, but re: your comment from email that deleting `search-an-address.js` breaks the site, I didn't find that - what part did it break?  Webpack complained because it was being imported in `index.js`, but once I removed that orphaned import everything seemed to work fine.